### DISCUSSION TITLE
ACP-125: Reduce C-Chain minimum base fee from 25 nAVAX to 1 nAVAX

### DISCUSSION THREAD

**Discussion by avalanche-foundation-admin** [avalanche-foundation/ACPs#127] [Discussions]
Proposes reducing the minimum base fee on the Avalanche C-Chain from 25 nAVAX to 1 nAVAX.

https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/125-basefee-reduction

**Comment by BaalMarduk** [comment]
I strongly support ACP 125 to reduce the C-Chain minimum base fee from 25 nAVAX to 1 nAVAX. Here are a few reasons why this change is beneficial:

	1.	Market Efficiency: The current minimum base fee of 25 nAVAX is higher than what the market demands, leading to artificially reduced network usage. By lowering the minimum base fee to 1 nAVAX, we allow the market to dictate the appropriate fee levels, resulting in a more efficient allocation of resources and potentially higher network usage.
	2.	Accessibility: Lowering the base fee reduces the cost barrier for users, making the Avalanche network more accessible to a broader audience. This inclusivity can drive adoption, foster innovation, and encourage more transactions, benefiting the entire ecosystem.
	3.	Dynamic Fee Mechanism: Historical data, such as the network’s response during high-usage periods in December 2023, has shown that the dynamic fee algorithm can handle fluctuations effectively. This gives us confidence that a lower base fee will not compromise the network’s stability or performance.
	4.	Network Growth: Reduced transaction costs can stimulate network activity and growth. As more users and developers are drawn to the Avalanche network due to lower fees, we can expect increased usage, more dApps, and a vibrant, expanding ecosystem.
	5.	Price Arbitrage: Lowering the base fee helps market makers perform more transactions profitably, facilitating price arbitrage. This can lead to more efficient markets, as market makers can capitalize on smaller price discrepancies, ultimately benefiting the entire network.
	6.	Competitiveness: Reducing the base fee will make the Avalanche C-Chain more competitive with other networks like Solana, which are known for their low transaction costs. This can attract more users and projects to Avalanche, further strengthening its position in the blockchain space.

**Comment by discochuck** [comment]
Made a dashboard to visualize what the difference you've spent on gas in avax terms and usd equiv terms would be had ACP125 been ACP125'n this whole time:

https://dune.com/couchdicks/avax-spent-on-gas
